contents/tutorials/how-to-rename-events.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: How to rename events
+sidebar: Docs
+showTitle: true
+author:
+  - bijan-boustani
+date: 2024-10-22
+tags:
+  - actions
+  - events
+---
+
+**Estimated reading time:** 3 minutes ☕☕
+
+**tl;dr** Use [Actions](/docs/data/actions). Start capturing events with the new name, and then create an Action to group it with the existing event.
+
+## Capturing events
+
+Once your PostHog account is up and running, you'll start capturing events. Some events are [autocaptured](/docs/product-analytics/autocapture). And you can also [capture custom events](/docs/product-analytics/capture-events).
+
+You can find all your events in the [Data management](/data-management/actions) tab in the left navigation.
+
+But once an event is captured, there is no option to rename it!
```

Comment:
I dont think this intro adds much value. Instead, I think we should write something on scenarios when you probably want to rename events/ use this